x-pack/plugins/rollup/server/lib/search_strategies/rollup_search_strategy.js

```diff
@@ -0,0 +1,36 @@
+/*
+* Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+* or more contributor license agreements. Licensed under the Elastic License;
+* you may not use this file except in compliance with the Elastic License.
+*/
+import { callWithRequestFactory } from '../call_with_request_factory';
+
+const ROLLUP_INDEX_CAPABILITIES_METHOD = 'rollup.rollupIndexCapabilities';
+const INDEX_PATTERN_SEPARATOR = ',';
+
+export default (AbstractSearchStrategy, RollupSearchRequest) =>
+  (class RollupSearchStrategy extends AbstractSearchStrategy {
+    name = 'rollup';
+    batchRequestsSupport = false;
+
+    constructor(server) {
+      super(server, callWithRequestFactory, RollupSearchRequest);
+    }
+
+    async isRollupJobExists(req, indexPattern) {
+      const callWithRequest = this.getCallWithRequestInstance(req);
+      const indices = (indexPattern || '').split(INDEX_PATTERN_SEPARATOR);
+
+      const requests = indices.map(index => callWithRequest(ROLLUP_INDEX_CAPABILITIES_METHOD, {
+        indexPattern: index,
+      }));
+
+      return Promise.all(requests)
+        .then((responses) => responses.some((response, index) => Boolean(response[indices[index]])))
```

Comment:
I think that @jen-huang told us that to use the rollup search we need to have at maximum 1 rollup index on the indexpattern. It will not work if the user wrote an index-pattern string that includes 2 rollup indexes. @jen-huang correct me if I'm wrong

Reply:
From our requirements: 

> Based on the input in Panel Options in the Index Pattern field, TSVB will be able to query:
> - regular indices
> - rollup index 
> - index combining a rollup and index pattern

I need this one for add supports of:  index combining a rollup and index pattern

Reply:
@markov00 @alexwizp Correct, the rollup search api only supports a maximum of one rollup index, along with any number of regular indices. See the 'rules' section in docs: https://www.elastic.co/guide/en/elasticsearch/reference/master/rollup-search.html#_path_parameters_69

This method could be changed to `numberOfRollupJobs()`, and returning the number of rollup index matches. `isViable()` can then check if `numberOfRollupJobs() === 1`.

Reply:
Got it, will be fixed